packages/cli/test/commands/routes.test.ts

```diff
@@ -0,0 +1,27 @@
+const routes = jest.fn(() => {
+  return []
+})
+jest.mock("@blitzjs/server", () => ({routes, resolveBinAsync: jest.fn()}))
+
+let onSpy: jest.Mock
+const spawn = jest.fn(() => {
+  onSpy = jest.fn(function on(_: string, callback: (_: number) => {}) {
+    callback(0)
+  })
+  return {on: onSpy, off: jest.fn()}
+})
+
+jest.doMock("cross-spawn", () => ({spawn}))
+
+import {Routes} from "../../src/commands/routes"
```

Comment:
Nit: Should we move this import to the top of the file?

Reply:
Will do!